src/App.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { v4 as uuidv4 } from 'uuid';
 
```

Comment:
'npm i -S uuid' try this to fix lint errors

Reply:
@StasSokolov
 I did this, but still have problem with the test